src/harness/audio/miniaudio.c

```diff
@@ -175,15 +184,19 @@ int AudioBackend_SoundIsPlaying(void* type_struct_sample) {
     return 0;
 }
 
-tAudioBackend_error_code AudioBackend_SetVolume(void* type_struct_sample, int volume_db) {
+tAudioBackend_error_code AudioBackend_SetVolume(void* type_struct_sample, int volume) {
     tMiniaudio_sample* miniaudio;
     float linear_volume;
 
     miniaudio = (tMiniaudio_sample*)type_struct_sample;
     assert(miniaudio != NULL);
 
-    // convert from directsound -10000 - 0 decibel volume scale
-    linear_volume = ma_volume_db_to_linear(volume_db / 100.0f);
+    if (!miniaudio->initialized) {
+        miniaudio->init_volume = volume;
+        return eAB_success;
+    }
+
+    linear_volume = volume / 510.0f;
```

Comment:
@b-kurczynski with these changes, does the spatial audio sound to you like the dos original game?